packages/repository-json-schema/test/integration/build-schema.integration.ts

```diff
@@ -258,124 +421,6 @@ describe('build-schema', () => {
-        @model()
-        class TestModel {
-          @property.array(Number)
-          numArr: number[];
-        }
-
-        const jsonSchema = modelToJsonSchema(TestModel);
-        expect(jsonSchema.properties).to.deepEqual({
-          numArr: {
-            type: 'array',
-            items: {
-              type: 'number',
-            },
-          },
-        });
-        expectValidJsonSchema(jsonSchema);
-      });
-
-      it('properly converts custom type arrays properties', () => {
-        class CustomType {
-          prop: string;
-        }
-
-        @model()
-        class TestModel {
-          @property.array(CustomType)
-          cusArr: CustomType[];
-        }
-
-        const jsonSchema = modelToJsonSchema(TestModel);
-        expect(jsonSchema.properties).to.deepEqual({
-          cusArr: {
-            type: 'array',
-            items: {
-              $ref: '#/definitions/CustomType',
-            },
-          },
-        });
-        expectValidJsonSchema(jsonSchema);
-      });
-
-      it('supports explicit primitive type decoration via strings', () => {
-        @model()
-        class TestModel {
-          @property({type: 'string'})
-          hardStr: Number;
-          @property({type: 'boolean'})
-          hardBool: String;
-          @property({type: 'number'})
-          hardNum: Boolean;
-        }
-
-        const jsonSchema = modelToJsonSchema(TestModel);
-        expect(jsonSchema.properties).to.deepEqual({
-          hardStr: {
-            type: 'string',
-          },
-          hardBool: {
-            type: 'boolean',
-          },
-          hardNum: {
-            type: 'number',
-          },
-        });
-        expectValidJsonSchema(jsonSchema);
-      });
-
-      it('maps "required" keyword to the schema appropriately', () => {
-        @model()
-        class TestModel {
-          @property({required: false})
-          propOne: string;
-          @property({required: true})
-          propTwo: string;
-          @property()
-          propThree: number;
-        }
-
-        const jsonSchema = modelToJsonSchema(TestModel);
-        expect(jsonSchema.required).to.deepEqual(['propTwo']);
-        expectValidJsonSchema(jsonSchema);
-      });
-
-      it('errors out when explicit type decoration is not primitive', () => {
-        @model()
-        class TestModel {
-          @property({type: 'NotPrimitive'})
-          bad: String;
-        }
-
-        expect(() => modelToJsonSchema(TestModel)).to.throw(/Unsupported type/);
-      });
-
-      it('errors out when "@property.array" is not used on an array', () => {
-        @model()
-        class BadArray {
-          @property()
-          badArr: string[];
-        }
-
-        expect(() => {
-          modelToJsonSchema(BadArray);
-        }).to.throw(/type is defined as an array/);
-      });
-
-      it('errors out if "@property.array" is given "Array" as parameter', () => {
```

Comment:
Why are all these tests removed?

Reply:
These tests have been moved above for better organization, I'll quickly check if I actually removed any of the tests